app/components/CollapsibleDisplayContent/index.tsx

```diff
@@ -0,0 +1,92 @@
+import Editor from '@webkom/lego-editor';
+import '@webkom/lego-editor/dist/style.css';
+import 'react-image-crop/dist/ReactCrop.css';
+import { type CSSProperties, useRef, useState } from 'react';
+import Icon from '../Icon';
+import styles from './CollapsibleDisplayContent.css';
+
+type Props = {
+  /** The content to be displayed - the text */
+  content: string;
+
+  /** The id of the div wrapping the content - the id */
+  id?: string;
+
+  /** The classname of the div wrapping the content - the className */
+  className?: string;
+
+  /** Any style tp be added to the div wrapping the content - the style */
```

Comment:
Although comments are often nice, I'm more a fan of not commenting on props, as I feel they are self explanatory. If you need to explain the props, that just means the naming is bad.

Also there's a typo on the last one; `tp -> to`.

Reply:
I agree. Removed them from `DisplayContent as well`